package.json

```diff
@@ -28,21 +30,30 @@
     }
   },
   "devDependencies": {
+    "@types/chai": "^4.2.10",
+    "@types/mocha": "^7.0.1",
     "@rollup/plugin-node-resolve": "^7.1.1",
     "@typescript-eslint/eslint-plugin": "^2.21.0",
     "@typescript-eslint/parser": "^2.21.0",
+    "chai": "^4.2.0",
+    "cross-env": "^7.0.0",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-prettier": "^3.1.2",
+    "fast-xml-parser": "^3.16.0",
     "husky": "^4.2.3",
+    "json2typescript": "^1.2.5",
```

Comment:
⚠️ this will be required at runtime. It is ok for now as we run only test with the code introduced here